joomla/joomla-cms
# PR for issue #36988 Rename "Send Mail to Administrators"

Pull Request for Issue #36988 .

### Summary of Changes
I changed 
COM_USERS_CONFIG_FIELD_MAILTOADMIN_LABEL="Send Mail to Administrators"

to 

COM_USERS_CONFIG_FIELD_MAILTOADMIN_LABEL="Send System Mail"


### Testing Instructions
Install the Patch, in User Options, this field should be renamed just like in this PR for 3.10.x
https://github.com/joomla/joomla-cms/pull/36989


### Actual result BEFORE applying this Pull Request
Misleading label, the Mail is not only towards Administrators
